recipe/meta.yaml

```diff
@@ -13,15 +13,15 @@ source:
     - tzdata-location.patch
 
 build:
-  number: 5
-  skip: true  # [py>39]
+  number: 0
+  skip: true  # [py>38]
```

Comment:
I think skippping `py<39` is correct. But it's interesting if we can build without skipping at all. Or then it will fail?

Reply:
According to `setup.cfg`, `py38` is the latest that is supported: https://github.com/pganssle/zoneinfo/blob/master/setup.cfg#L23

The package is just not necessary for `py39` and later because `zoneinfo` is its own package: https://docs.python.org/3/library/zoneinfo.html